### PR TITLE
Adding an extra cell to the bottom right corner for padding in the graph

### PR DESCRIPTION
**Pull request by aasimkhan30**
This fixes the corner edges getting truncated in bottom and left corners

Before:
image

After:
image

### CODE REVIEW

File: src/js/azdata/azdataQueryPlan.js [hunk 1]
```diff
@@ -362,7 +371,8 @@ azdataQueryPlan.prototype.init = function (container, iconPaths) {
                 }
             }
         }
-        //layout.execute(parent);
+        // Adding a very small cell to the parent for padding on the bottom right corner of the graph. 
+        vertex = graph.insertVertex(parent, 'paddingVertex', undefined, maxX+70+100, maxY+70+100, 0.0001, 0.0001, '');
```
Comment (lewis-sanchez) [inline src/js/azdata/azdataQueryPlan.js]: Why are calculations like `maxX+70+100` broken down like that? Seems simpler to just make it `maxX+170`. I'd also assign 170 to a const with an appropriate name, so it's easier to understand what it's meaning is.

Comment (aasimkhan30) [reply src/js/azdata/azdataQueryPlan.js]: 70 is the cell width and 100 is the padding. I will add 2 const variables for it.

File: src/js/azdata/azdataQueryPlan.js [hunk 2]
```diff
@@ -340,6 +340,9 @@ azdataQueryPlan.prototype.init = function (container, iconPaths) {
                     node: this.queryPlanGraph
                 }
             ];
+
+        var maxX = -1;
+        var maxY = -1;
         while (stack.length > 0) {
             var entry = stack.pop();
             if (entry.node.children) {
```
Comment (kburtram) [inline src/js/azdata/azdataQueryPlan.js:356]: the max calculations still work if the execution plan is a single node (i.e. without children)?  I forgot whether this condition is guaranteed to be true at least once for all inputs.

**Review by lewis-sanchez — APPROVED** [review]

**Review by kburtram — APPROVED** [review]
LGTM.  I think adding constant variables for the constant values makes sense, and there is a possible corner-case when the graph has a single element (but don't recall data structure details enough to confirm).

File: src/js/azdata/azdataQueryPlan.js (hunk 2 above)
Comment (aasimkhan30) [reply src/js/azdata/azdataQueryPlan.js:356]: Good catch. While I have never seen a plan with a single node but it does not hurt to handle that case over here.

**Pull request merged**